docs/install/Knative-with-Kourier.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Installing Knative with Kourier"
+linkTitle: "Kourier Ingress and Knative"
+weight: 15
+type: "docs"
+---
+
+[Kourier](https://github.com/3scale/kourier) is an open-source lightweight Knative Ingress based on Envoy. It's been designed for Knative, without requiring any additional custom resource definitions (CRDs).
+
+This guide walks you through the installation of the latest version of Knative
+with Kourier as the ingress.
+
+## Before you Begin
+
+Knative requires a Kubernetes cluster v1.14 or newer, as well as a compatible `kubectl`. This guide assumes that you have already [created a Kubernetes cluster](https://kubernetes.io/docs/setup/) and are using
+bash in a Mac or Linux environment.
+
+## Install Knative
+
+Let's do a core install of Knative Serving with the released yaml templates:
+
+1.  To install Knative, first install the CRDs by running the following `kubectl apply`
```

Comment:
We just got automatic numbering to render properly on the website! Lists that are indented at the same level will auto-number for you:

```
1. one

1. two

1. three
```

becomes

1. one

1. two

1. three

Reply:
(so the fix here is just to start every numbered list item with "1.")